modin/test/test_partition_api.py

```diff
@@ -19,14 +19,38 @@
 from modin.distributed.dataframe.pandas import unwrap_partitions, from_partitions
 from modin.config import Engine, NPartitions
 from modin.pandas.test.utils import df_equals
+from modin.pandas.indexing import compute_sliced_len
+from modin.data_management.factories.dispatcher import EngineDispatcher
 
+PartitionClass = (
+    EngineDispatcher.get_engine().io_cls.frame_cls._partition_mgr_cls._partition_class
+)
 
 if Engine.get() == "Ray":
     import ray
-if Engine.get() == "Dask":
+
+    put_func = ray.put
+    get_func = ray.get
+    FutureType = ray.ObjectRef
+elif Engine.get() == "Dask":
     from distributed.client import get_client
+    from distributed import Future
+
+    put_func = lambda x: get_client().scatter(x)  # noqa: E731
```

Comment:
@dchigarev , did you use lambda to scatter data under this assumption from dask distributed documentation?
```
Note that it is often better to submit jobs to your workers to have them load the data rather than loading data locally and then scattering it out to them.
```

Reply:
I think that it's somewhat irrelevant as this is for testing only, and that note is mostly about performance.